modules/ROOT/pages/configure-headers4-task.adoc

```diff
@@ -1,37 +1,73 @@
-= To Configure Headers
+= Configuring the Response Headers and Status Code
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In the HTTP Listener, you can configure a verbose form of the response HTTP headers and status code. You configure two properties:
+In the *HTTP Listener* configuration, you can customize the HTTP response body, headers, status code, and reason phrase.
+
+== Configure Variable Names
+
+You can configure the variables that the HTTP listener searches for by default by configuring the following two properties in *APIkit Router*:
```

Comment:
```suggestion
You can configure the default variables that the HTTP listener searches for by configuring the following two properties in *APIkit Router*:
```

Reply:
(not sure if that changes the meaning, but the "by default" sounds awkward in the original.)

Reply:
The "by default by" sounds weird!
Thanks for solving it! :)